tb_plugin/torch_tb_profiler/profiler/data.py

```diff
@@ -34,7 +34,9 @@ def __init__(self, worker):
         self.data_schema_version = None
         self.events = None
         self.trace_file_path = None
-        self.is_gpu_used = False
+        self.has_runtime = False
+        self.has_kernel = False
+        self.has_memory = False
```

Comment:
rename `has_memory`

Reply:
Thanks. Renamed to "has_memcpy_or_memset"